components/x-gift-article/src/lib/share-link-actions.js

```diff
@@ -0,0 +1,40 @@
+function createMailtoUrl (articleTitle, shareUrl) {
+	const subject = encodeURIComponent(articleTitle);
+	const body = encodeURIComponent(shareUrl);
+
+	return `mailto:?subject=${subject}&body=${body}`;
+}
+
+
+function copyToClipboard (event) {
+
+	const urlSection = event.target.closest('fieldset > .js-gift-article__url-section');
```

Comment:
I'm struggling to parse this... is it trying to find the closest fieldset parent element or element matching the class name? Either way, is it possible to avoid the combinator?

Reply:
🙀 Sorry! the combinator is unnecessary!!